tiledb/sm/subarray/subarray.cc

```diff
@@ -1011,7 +1011,20 @@ Status Subarray::compute_tile_overlap() {
         } else {  // Sparse fragment
           auto rtree = (const RTree*)nullptr;
           RETURN_NOT_OK(meta[i]->rtree(*encryption_key, &rtree));
-          tile_overlap_[i][j] = rtree->get_tile_overlap<T>(range);
+          if (rtree != nullptr) {  // Format version > 2
```

Comment:
should this be an explicit version check instead of an implicit null check?

Reply:
I can do this change.